3.distributed-calculator/README.md

```diff
@@ -9,7 +9,85 @@ This sample shows method invocation and state persistent capabilities of Dapr th
 
 The front-end application consists of a server and a client written in [React](https://reactjs.org/). 
 Kudos to [ahfarmer](https://github.com/ahfarmer) whose [React calculator](https://github.com/ahfarmer/calculator) 
-sample was used for the client. The following architecture diagram illustrates the components that make up this sample: 
+sample was used for the client.
+
+## Running locally
+
+We will be starting the four operator apps (add, subtract, multiply and divide) along with the dapr sidecar locally and then run the front end app which persists the state in a local redis state store.
+
+1. Add App - Open a terminal window and navigate to the go directory and follow the steps below:
+   - Install the gorilla/mux package: Run:
+      ```
+      go get -u github.com/gorilla/mux
+      ```
+    - Build the app. Run:
+      ```
+      go build app.go
+      ```
+    - Run dapr using the command:
+      ```
+      dapr run --app-id addapp --app-port 6000 --port 3503 ./app
+      ```
+2. Subtract App - Open a terminal window and navigate to the csharp directory and follow the steps below:
+    - Set environment variable to use non-default app port 7000
+      ```
+      Linux/Mac OS:
+      export ASPNETCORE_URLS=7000
+      
+      Windows:
+      set ASPNETCORE_URLS=7000
+      ```
+    - Build the app. Run:
+      ```
+      dotnet build
+      ```
+    - Navigate to ./bin/Debug/netcoreapp3.1 and start Dapr using command:
+      ```
+      dapr run --app-id subtractapp --app-port 7000 --port 3504 dotnet Subtract.dll
+      ```
+3. Divide App - Open a terminal window and navigate to the node directory and start Dapr using the command below:
+    ```
+    dapr run --app-id divideapp --app-port 4000 --port 3502 node app.js
```

Comment:
Let's add a step to install dependencies using `npm install` before this step.

Reply:
Fixed